### DISCUSSION TITLE
Broken project website URL (1600 components)

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5907] [Q&A] [answered]
The project site was offline, how to get rid of the alert in bulk now? I want to get rid of it completely, not mark it dismiss.
There is information in the documentation that the alerts will disappear after the problem is solved.
![image](https://user-images.githubusercontent.com/39633242/115866185-9e3fbb00-a439-11eb-8302-6cdf371c73cc.png)

**Comment by tomkolp** [comment, marked as answer]
After 24h, the alert will disappear.

**Reply by nijel** [reply]
Yes, the alerts are updated daily, or when related change happens. I've clarified the docs in 2f131c0205.
